client/src/components/Login/Login.js

```diff
@@ -44,8 +44,12 @@ const Login = () => {
 
   const navigate = useNavigate();
 
-  const clearData = () => {
-    setLoginData({ email: "", password: "" });
+  const clearData = (shoulClearAll) => {
```

Comment:
I think you misspelled *should* for this variable here and in `Register.js`.

Reply:
oh my bad , Correcting this one. Apologies for this one.